src/dev/performance/run_performance_cli.ts

```diff
@@ -33,19 +108,35 @@ run(
       throw createFlagError('--journey-path must be an existing path');
     }
 
-    let journeys: Journey[] = [];
+    const journeys: Journey[] = [];
 
     if (journeyPath) {
-      journeys = fs.statSync(journeyPath).isDirectory()
-        ? fs.readdirSync(journeyPath).map((fileName) => {
-            return { name: fileName, path: path.resolve(journeyPath, fileName) };
+      const isDirectory = fs.statSync(journeyPath).isDirectory();
+      if (isDirectory) {
+        const files = fs.readdirSync(journeyPath);
+        journeys.push(
+          ...files.map((file) => {
+            return {
+              name: path.parse(file).name,
+              path: path.resolve(journeyPath, file),
+            };
           })
-        : [{ name: path.parse(journeyPath).name, path: journeyPath }];
+        );
+      } else {
+        journeys.push({ name: path.parse(journeyPath).name, path: journeyPath });
+      }
     } else {
+      // reading all existing journeys
       const journeyBasePath = path.resolve(REPO_ROOT, JOURNEY_BASE_PATH);
-      journeys = fs.readdirSync(journeyBasePath).map((name) => {
-        return { name, path: path.join(journeyBasePath, name) };
-      });
+      journeys.push(
+        ...fs.readdirSync(journeyBasePath).map((file) => {
+          return { name: path.parse(file).name, path: path.join(journeyBasePath, file) };
+        })
```

Comment:
similar to my comment above: this doesn't seem to support sub-directories. Is that intended?

Reply:
At the moment we don't have sub-directories, but I think you pointed a reasonable concern. I will make a change

Reply:
Fixed in f520b3a957dd2abbd93395413f06ce172e3fd6c9

I copied journey to newly create `test1` sub-folder:
<img width="1066" alt="Screenshot 2024-02-14 at 12 36 29" src="https://github.com/elastic/kibana/assets/10977896/2f7d277c-9ef6-4ecd-b2d9-28dd7a1c1408">